mmv1/templates/terraform/examples/sql_mysql_instance_private_ip.tf.erb

```diff
@@ -36,8 +36,9 @@ resource "google_sql_database_instance" "<%= ctx[:primary_resource_id] %>" {
   settings {
     tier = "db-f1-micro"
     ip_configuration {
-      ipv4_enabled    = "false"
-      private_network = google_compute_network.private_network.id
+      ipv4_enabled                                  = "false"
```

Comment:
Similar comment as above: I suspect these examples are not being used for anything.

Reply:
I have reverted this unnecessary edit and created an issue ticket to this issue: https://github.com/hashicorp/terraform-provider-google/issues/13647